spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -31,29 +33,27 @@
     it 'redirects logged in user to the prompt' do
       request_prompt = create(:prompt, collection_id: collection.id, challenge_signup_id: signup.id)
       claim_with_prompt = create(:challenge_claim, collection: collection, request_prompt_id: request_prompt.id)
-      fake_login_known_user(user)
-      get :show, id: claim_with_prompt.id, collection_id: collection.name
+      get :show, id: claim_with_prompt, collection_id: collection.name
       it_redirects_to(collection_prompt_path(collection, claim_with_prompt.request_prompt))
     end
 
-    xit 'needs a collection' do
-      fake_login_known_user(user)
-      get :show
-      it_redirects_to_with_error(root_path, \
-                                 "What challenge did you want to work with?")
+    it 'redirects logged in users if no collection is given' do
```

Comment:
I'm not sure this is right. Contrary to what this says, it looks like a collection _is_ being specified (`get :show, id: 999_999, collection_id: collection.name`), plus the error message seems to think the _claim_ is the problem. To me, it looks like `it "redirects logged in users if the claim does not exist"`.